### PR TITLE
Fixing git history

### PR DESCRIPTION
**Pull request by mfeigl**
### Description

**TL;DR** `main` and `v.next` are identical in terms of their code; however, their histories differ and think there things to merge into each other: [main...v.next](https://github.com/Esri/arcgis-maps-sdk-java-samples/compare/main...v.next) and [v.next...main](https://github.com/Esri/arcgis-maps-sdk-java-samples/compare/v.next...main). I fixed `v.next` but `main` is protected and need this PR to reconcile its history. 

**Here are the steps I performed:**
$ git checkout main // checkout main
$ git merge v.next // merge v.next into main
$ git checkout v.next // checkout v.next
$ git merge main // merge main into v.next
$ git push origin HEAD // push to v.next - makes no changes as the two branches are identical, but reconciles the history
$ git checkout main // checkout main
$ git checkout -b mfeigl/main // main is protected, so I need a new branch (this one)
$ git push origin HEAD // push to this branch

As you can see, [mfeigl/main...v.next](https://github.com/Esri/arcgis-maps-sdk-java-samples/compare/mfeigl/main...v.next) and [v.next...mfeigl/main](https://github.com/Esri/arcgis-maps-sdk-java-samples/compare/v.next...mfeigl/main) are identical and there are no differences between the two branches.

I already pushed onto `v.next` now I just need to push this branch onto `main`, and then the two branches will have the same history. 

Checklist:
- [x] Set target branch to main (current release) or v.next (next release)
- [x] Request a reviewer
- [x] Assign a reviewer
- [x] Tag reviewer in comment

### CODE REVIEW

**Comment by mfeigl** [comment]
@D-R-Smith could you please review this PR?

**Review by D-R-Smith — APPROVED** [review]
@mfeigl looks okay to me

**Comment by mfeigl** [comment]
@jenmerritt could you please have a look at this PR? This is a follow up on the previous one which made sure the codes are identical between the two branches and then this one makes sure their histories are identical as well.

**Review by jenmerritt — APPROVED** [review]
Go for it
